### PR TITLE
Generate an all-in-one nuget package.

### PR DESCRIPTION
**Pull request by Akarinnnnn**
Added a nuspec for all-in-one package and instructions to pack such package.
But alternatively, `dotnet pack` is broken.

### CODE REVIEW

**Comment by Akarinnnnn** [comment]
It's normal to see a lot of stuff in the `bin/rutimes` folder if you referenced this package. Publishing your app to a specific platform, such as `win-x64` will pick one suitable dll to the output folder.

File: Standalone/How to build nuget package.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+```bat
+dotnet build -t:BatchBuild Steamworks.NET.Standard.sln
+```
+to your terminal.
+
+Then use nuget client with the `.nuspec` to pack an all-in-one package. Enter
+```bat
+nuget pack Steamworks.NET.nuspec -OutputDirectory bin\
+```
+to your terminal.
```
Comment (rlabrecque) [inline Standalone/How to build nuget package.md:11]: Do you think we could have a build.bat and build.sh which runs these?

File: Standalone/Steamworks.NET.Standard.csproj [hunk 2]
```diff
@@ -1,200 +1,67 @@
...
-    <Compile Include="..\com.rlabrecque.steamworks.net\Runtime\types\Steam_api_common\HSteamPipe.cs" Link="Steamworks.NET\types\SteamClient\HSteamPipe.cs" />
-    <Compile Include="..\com.rlabrecque.steamworks.net\Runtime\types\Steam_api_common\HSteamUser.cs" Link="Steamworks.NET\types\SteamClient\HSteamUser.cs" />
-  </ItemGroup>
-
-  <Target Name="Copy Nuget packages" AfterTargets="Pack">
-    <Copy SourceFiles="$(outdir)..\$(PackageId).$(packageversion).nupkg" DestinationFolder="$(projectdir)/bin/Nuget Packages/" />
-  </Target>
+	<ItemGroup>
+		<Compile Include="../com.rlabrecque.steamworks.net/Runtime/**/*.cs" />
+	</ItemGroup>
```
Comment (rlabrecque) [inline Standalone/Steamworks.NET.Standard.csproj:64]: Nice!

File: Standalone/How to build nuget package.md (hunk 1 above)
Comment (Akarinnnnn) [reply Standalone/How to build nuget package.md:11]: Sure!

**Pull request merged**